.github/workflows/build.yaml

```diff
@@ -46,4 +46,17 @@ jobs:
           cargo fmt --verbose --check -- --color always
       - name: Run clippy
         run: |
-          cargo clippy --all-features --all-targets --color always -- --deny warnings
\ No newline at end of file
+          cargo clippy --all-features --all-targets --color always -- --deny warnings
+  python-lint:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout source code
+        uses: actions/checkout@v3
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: '3.9'
+      - name: Install black
+        uses: psf/black@stable
+        with:
+          args: "watchtower-plugin/tests --check -l 120"
```

Comment:
These are called `options` not `args`.
Also there is a dedicated parameter called `src` which I guess selects what sources you wanna format.
as per `https://black.readthedocs.io/en/stable/integrations/github_actions.html`

Reply:
I see, using dedicated parameters is the better approach. I'll change args to options and specify the src in the next commit.